user-guide/Cloud_Platform/Connecting_to_cloud/Connect_to_dataminer_services.md

```diff
@@ -60,7 +60,7 @@ Before you connect your DMS to dataminer.services (unless you [connect with a DM
 
 1. Select the checkbox to agree to the terms of service and click *Connect*.
 
-1. On the System Center \> *Cloud* page, wait until the status under *Cloud info* changes to *Connected*. This can take up to half a minute.
+1. On the System Center \> *Cloud* page, wait until the status under *Registration status* changes to *Registered*. This can take up to half a minute.
```

Comment:
Is this change to *Registration status* implemented by RN 38715? We'll need to make sure the documentation is correct for people with older supported DataMiner versions as well, as there is only one version of the documentation for all DataMiner users.

Reply:
Yes indeed, but the 'Cloud info' was already wrong, that doesn't exist right now either. 
It's definitely worth adding now that currently it says Status: Connected' when it's done depending on the version, but will indeed be changed to 'Registration status: Registered'.